tests/gh.zunit

```diff
@@ -5,36 +5,79 @@
     load _helpers/mock.zsh
     load _helpers/assertions.zsh
     mock gh
+    mock __fzf_extract_command
 }
 
 @teardown {
     unmock gh
+    unmock __fzf_extract_command
 }
 
 @test 'Testing completion: gh **' {
     _fzf_complete() {
         fail '_fzf_complete should not be invoked'
     }
 
+    __fzf_extract_command_mock_1() {
+        assert $# equals 1
+        assert $1 same_as 'gh '
+
+        echo 'gh'
+    }
+
     _fzf_path_completion() {
         assert $# equals 2
         assert $1 same_as ''
         assert $2 same_as 'gh '
+
+        assert __fzf_extract_command mock_times 1
     }
 
     prefix=
     _fzf_complete_docker 'gh '
```

Comment:
I found the wrong function... In this file, there are some of them.

Reply:
What are they?

Reply:
They are what I forgot to edit after copy-pasted.